data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3483 @@
+					decline
+			`	He looks at you gravely, "Your loss. Do not come here crying when the Hai are causing you difficulties."`
+			choice
+				`	"Wait, I have changed my mind!"`
+				`	(Go.)`
+					decline
+			`	He makes a show of thinking it over before responding, "Well, I guess I can forgive your insolence, this time. But listen to what I am about to say."`
+			label agree
+			`	"Go to our home system and make your request, the Chiefs despise conflict, so they will most likely grant it. Given the circumstances, Hai-home will definitely not let you land though, so go to Mirrorlake instead."`
+				accept
+	on complete
+		conversation
+			`You find the Hai leaders that agreed to meet you and express your request to settle the disputes between you. They consider this amongst themselves before responding, "Tell us, <last>, captain of the <ship>, why should we grant you such a request?"`
+			choice
+				`	"Because otherwise you get shot!" (Draw your weapon)`
+					goto death
+				`	"I know I have done stupid things in the past, but I have learned my error. I promise not to make such mistakes again."`
+				`	"I can continue bringing food to the Unfettered for you, so they do not starve to death."`
+			`	They seem willing to give you a second chance. "You have convinced us, but we will not grant you this pardon twice."`
+				accept
+			label death
+			`	The leaders gasp, and then, in a moment that seems to last forever, there is silence whilst everyone is looking at you. The silence is shattered when the guards start shooting. They probably did not intend to kill you outright, but apparently they see you as enough of a threat that the result is the same.`
+				die
+		"reputation: Hai" >?= 1
+		log `Convinced the Hai to stop their conflict with me, as request by an Unfettered Chief.`
+
+
+
+mission "Unfettered: Bring Hai"
+	name `Bring True Hai back`
+	description `Bring <bunks> Hai from <stopovers> to <origin>.`
+	source "Firelode"
+	stopover "Snowfeather"
+	passengers 10
+	blocked "You will need room for 10 passengers in order to take the next Unfettered mission."
+	to offer
+		has "Unfettered Jump Drive 1: offered"
+		or
+			not "Unfettered: offered bring hai"
+			random < 20
+	on defer
+		set "Unfettered: offered bring hai"
+	on offer
+		conversation
+			`Looking outside you see the sun is setting... It is a breathtaking view, revealing a big white structure in the distance. As you exit your ship you notice movement. You are partly carried by a crowd of Unfettered that seem to be heading to a big, yet simple, square-shape stadium nearby. That building is grandiose compared to its surroundings, by its sheer length and pure white appearance. In contrast, most buidlings around it appear to be bars or houses, but they are made from less pure materials, or maybe they just have not been cleaned in a long time. Something of great interest must be happening in there, though you notice some Unfettered are just going to the bars around, chatting.`
```

Comment:
If the crowd of people is so large that the player is carried by them, then the player should notice more than movement when they exit their ship.
```suggestion
			`Looking outside you see the sun is setting... it is a breathtaking view that reveals a big white structure in the distance. As you exit your ship you notice movement. You are partly carried by a crowd of Unfettered that seem to be heading to a big, yet simple, square-shape stadium nearby. That building is grandiose compared to its surroundings, distinguished by its sheer length and pure white appearance. In contrast, most buidlings around it appear to be bars or houses, but they are made from less pure materials, or maybe they just have not been cleaned in a long time. Something of great interest must be happening in there, though you notice some Unfettered are just going to the bars around, chatting.`
```